src/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  margin: 0 16px 4px;
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__product-code {
+  margin-left: 16px;
```

Comment:
Let's specify correct general padding to the whole card instead of adding margin-left / right for all elements

Reply:
Ok